apps/strapi-cms/Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY --from=build /opt/node_modules ./node_modules
 WORKDIR /opt/app
 COPY --from=build /opt/app ./
 ENV PATH /opt/node_modules/.bin:$PATH
+RUN npm run configrestore
```

Comment:
What does this command do?

Reply:
"configrestore takes the configurations saved in viewconfig.json and copies them into the DB.
It currently does so with a 'merge' logic, as described [in the docs](https://docs.strapi.io/dev-docs/cli#strapi-configurationrestore), to avoid overwriting production configurations that may need to be kept."